tests/frontend/test_kibana_app/cypress/integration/features/management/settings/sample-data.feature

```diff
@@ -0,0 +1,31 @@
+Feature: add sample data to modules
+
+  As a kibana user
+  i want to add sample data to modules
+  in order to populate graphics
```

Comment:
maybe
```
As a Kibana user
I want to add sample data indices
in order to check modules
```

Reply:
yes, originally I was thinking of validating some of the graphics with that sample data, but then just validated that the toast messages are displayed, so that's what the feature should name.

maybe:
As a Kibana user
I want to add sample data indices
in order to check toast success messages.